src/main/java/net/dv8tion/jda/core/entities/Icon.java

```diff
@@ -112,9 +112,15 @@ public static Icon from(InputStream stream) throws IOException
      */
     public static Icon from(byte[] data)
     {
-        Args.notNull(data, "Provided byte[]");
+        Checks.notNull(data, "Provided byte[]");
 
-        String encoding = StringUtils.newStringUtf8(Base64.getEncoder().encode(data));
-        return new Icon(encoding);
+        try
+        {
+            return new Icon(new String(Base64.getEncoder().encode(data), "UTF-8"));
+        }
+        catch (UnsupportedEncodingException e)
+        {
+            throw new RuntimeException(e); // thanks JDK 1.4
+        }
```

Comment:
`org.apache.commons.codec.binary` is not available anymore since we removed Unirest and its dependencies.
This is essentially the same code though, just inlined.